store/postgres/src/relational_queries.rs

```diff
@@ -157,21 +157,29 @@ fn query_builder_error(msg: String) -> diesel::result::Error {
 }
 
 /// A `QueryValue` makes it possible to bind a `Value` into a SQL query
-/// where the needed SQL type is `ColumnType`
-struct QueryValue<'a>(&'a Value, ColumnType);
+/// using the metadata from Column
+struct QueryValue<'a>(&'a Value, &'a Column);
```

Comment:
I suppose if enums where a `ColumnType` this wouldn't need to change.

Reply:
You are right - it has to become a `&ColumnType` because `ColumnType` can't be `Copy` anymore, but yes, we don't need the whole column.